docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,133 @@
+Proctored Exams
+===============
+
+Helpful ReadTheDocs Pages:
+
+- `Configuring a Proctoring Provider <https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/proctored_exams/proctored_enabling.html#configuring-proctoring-provider>`_
+- `Creating a Proctored Exam <https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/proctored_exams/pt_create.html#creating-a-proctored-exam>`_
+
+Prerequisite steps:
+    #. Use Google Chrome
+    #. Have a staff account
+    #. Have a non-staff learner account enrolled in the verified track for this course
+    #. As your non-staff user in your current window, and enroll in the course where you have set up your proctored exam
+    #. In an incognito window, login to your staff account and visit your Support Tools page and search in the email for your non-staff user. You should now see the option to change the enrollment track of that user. Change the enrollment track of the user to ""verified"" (choose any reason when prompted, it doesn't matter)
+    #. In that same incognito, still signed in as staff, navigate to the Instructor dashboard
+    #. Find or create a test course within Stage with proctoring enabled and an LTI proctoring provider chosen, certificates are enabled, and where the exams IDA waffle flag is enabled
```

Comment:
I think that the order of these prerequisite steps is a little confusing. For example, you're enrolling a non-staff learner account in "this course" before you have set up a course with proctoring. Based on your instructions, you may set up an entirely new course in which the learner is not enrolled.

I'd recommend breaking this up into "Proctoring Set Up Steps" and "User Set Up Steps" to make this clearer.

Reply:
Do you have a particular environment in mind here (e.g. stage)? Stage will have proctoring set up in a way that someone just starting in their devstack will not, so I think we should either not make references to a particular environment and link out to documentation for each environment (e.g. stage/production and development), or specify that this is just for stage/production.

Reply:
"certificates are enabled" is a little vague and could mean a variety of things. I think we should be more specific here. Do you mean "where there is a verified track" or something else?

Reply:
I know we don't have official documentation for this yet, but it would be good to clarify what "an LTI proctoring provider chosen" means - how does one do this or verify this? Depending on which environment this documentation applies to, you could just refer to the Proctoring Settings Page and/or reference our [Local Development and LTI Configuration](https://2u-internal.atlassian.net/wiki/spaces/PT/pages/256737327/Local+Development+and+LTI+Configuration) document for the development environment.

Reply:
- Changed the order of the steps as you recommended (still need to push the change)
- I have local in mind, but based on [this thread](https://twou.slack.com/archives/C049EF6VCTB/p1699386718501819) it seems like it's also fair to mention that we do have a test course in stage (at least as of now) 
- I'm honestly not too sure how certificates work so if "verified track" makes more sense I can just put that.
- Assuming `edx-exams` ends up in Open edX, would it make sense to just move the instructions for [Local Development and LTI Configuration](https://2u-internal.atlassian.net/wiki/spaces/PT/pages/256737327/Local+Development+and+LTI+Configuration) to the README and just make that the source of truth?

Reply:
for the local instructions you are correct and those do need to move as part of the release to openedx. We've got that noted as a requirement here: https://2u-internal.atlassian.net/wiki/spaces/PT/pages/626753683/Beta+Release+and+Beyond.

I do think these tests need to be done in a deployed environment and not Tutor/Devstack in order to say we're confident in releasing these features.

Reply:
@MichaelRoytman I do think there's also going to be a gap until we get instructions in readthedocs. I don't think we should duplicate how to setup a proctored exam, a user, a proctoring provider, etc here and should just reference readthedocs where needed.

Reply:
Yeah, totally agree with linking out. I didn't mean to suggest we should duplicate - just that I felt like some information was missing, which can definitely be added by referencing other documentation, like "Set up a proctored exam: <link>."